src/utils/hooks/useLanguage.ts

```diff
@@ -0,0 +1,94 @@
+import { usePathname } from "next/navigation";
```

Comment:
I'd like some block comments for the hook 💡

Reply:
Added some now